dirsrvtests/tests/suites/webui/database/database_test.py

```diff
@@ -69,31 +69,52 @@ def test_global_database_configuration_availability(topology_st, page, browser_n
     setup_login(page)
     time.sleep(1)
     frame = check_frame_assignment(page, browser_name)
-
-    log.info('Check if element on Limits tab is loaded.')
-    frame.get_by_role('tab', name='Database', exact=True).click()
-    frame.get_by_text('ID List Scan Limit', exact=True).wait_for()
-    assert frame.get_by_text('ID List Scan Limit', exact=True).is_visible()
-
-    log.info('Click on Database Cache tab and check if element is loaded')
-    frame.get_by_role('tab', name='Database Cache', exact=True).click()
-    assert frame.locator('#db_cache_auto').is_visible()
-
-    log.info('Click on Import Cache tab and check if element is loaded')
-    frame.get_by_role('tab', name='Import Cache', exact=True).click()
-    assert frame.locator('#import_cache_auto').is_visible()
-
-    log.info('Click on NDN Cache tab and check if element is loaded')
-    frame.get_by_role('tab', name='NDN Cache', exact=True).click()
-    assert frame.get_by_text('Normalized DN Cache Max Size').is_visible()
-
-    log.info('Click on Database Locks tab and check if element is loaded')
-    frame.get_by_role('tab', name='Database Locks', exact=True).click()
-    assert frame.locator('#dblocksMonitoring').is_visible()
-
-    log.info('Click on Advanced Settings tab and check if element is loaded')
-    frame.get_by_role('tab', name='Advanced Settings', exact=True).click()
-    assert frame.locator('#txnlogdir').is_visible()
+    instance = topology_st.all_insts.get('standalone1')
```

Comment:
You can also call the instance by:
instance = topology_st.standalone

Otherwise LGTM

Reply:
Thanks, fixed this as it's more elegant.